### DISCUSSION TITLE
is it possible for us to remove photos from the album?

### DISCUSSION THREAD

**Discussion by LHansoon** [RhetTbull/osxphotos#448] [Q&A]
Hey guys, 

Do we have a way to remove photos from the album? Also, do we have a way to delete the photo?
Sorry in advance in case if I didn't find it.

Thanks!

**Comment by RhetTbull** [comment]
Hi. Osxphotos doesn't modify the Photos library (for example deleting photos or modifying albums). It's primary use is to export photos. Can you provide more details about what you want to do?  If you know a bit of Python you can use my other library photoscript (https://github.com/RhetTbull/PhotoScript) to modify the library including deleting photos from an album.

**Reply by LHansoon** [reply]
Thanks for the reply, I will take that a look!
